internals/daemon/daemon_test.go

```diff
@@ -510,10 +531,55 @@ func (s *daemonSuite) TestSuperAccess(c *C) {
 		write: AdminAccess{},
 	}}
 
-	for _, uid := range []int{0, os.Getuid()} {
-		remoteAddr := fmt.Sprintf("pid=100;uid=%d;socket=;", uid)
-		s.testAccessChecker(c, tests, remoteAddr)
+	s.testAccessChecker(c, tests, "pid=100;uid=0;socket=;")
+	s.testAccessChecker(c, tests, "pid=100;uid=1;socket=;") // adminuser
+	s.testAccessChecker(c, tests, fmt.Sprintf("pid=100;uid=%d;socket=;", os.Getuid()))
+}
+
+func (s *daemonSuite) TestDefaultUcredUsers(c *C) {
+	d := s.newDaemon(c)
+
+	var user *UserState
```

Comment:
Maybe call this `userSeenInCommand` (or something like that) to make it more obvious (if I got it right) that this is the user that was "seen" when the command was called?

Reply:
Fair enough. I've renamed to `userSeen`.